src/Native/Unix/System.Native/pal_time.c

```diff
@@ -193,7 +194,10 @@ int32_t SystemNative_GetCpuUtilization(ProcessCpuInformation* previousCpuInfo)
     int32_t cpuUtilization = 0;
     if (cpuTotalTime > 0 && cpuBusyTime > 0)
     {
-        cpuUtilization = (int32_t)(cpuBusyTime / cpuTotalTime);
+        if (asPercent)
+            cpuUtilization = (int32_t)(cpuBusyTime * 100 / cpuTotalTime);
```

Comment:
This looks like a bug to me. I think this should always multiply by `100`.

Reply:
That's what I initially thought but I wasn't sure who wrote this :-) @jkoritzinsky Could you verify? If it's a bug I can modify this PR to fix this instead of adding a new surface.

Reply:
Yep it's supposed to be multiplied by 100.

Looks like when I wrote this I got my wires crossed on whether or not my range was 0.0-1.0 or 0-100. The code that uses this expects it to be between 0-100 so feel free to add the multiplier.